tests/Api/TimestampShapeTest.php

```diff
@@ -22,7 +22,10 @@ public function formatProvider()
             [$t, 'rfc822', 'Tue, 05 Jan 1999 00:00:00 GMT'],
             [new \DateTime('january 5, 1999'), 'unixTimestamp', '915494400'],
             [new \DateTime('january 5, 1999'), 'iso8601', '1999-01-05T00:00:00Z'],
-            [new \DateTime('january 5, 1999'), 'rfc822', 'Tue, 05 Jan 1999 00:00:00 GMT']
+            [new \DateTime('january 5, 1999'), 'rfc822', 'Tue, 05 Jan 1999 00:00:00 GMT'],
+            [new \DateTimeImmutable('january 5, 1999'), 'rfc822', 'Tue, 05 Jan 1999 00:00:00 GMT'],
+            [new \DateTimeImmutable('january 5, 1999'), 'iso8601', '1999-01-05T00:00:00Z'],
+            [new \DateTimeImmutable('january 5, 1999'), 'rfc822', 'Tue, 05 Jan 1999 00:00:00 GMT']
```

Comment:
Isn't this the same as line 26? Supposed to be that way?

Reply:
Thanks!